main.js

```diff
@@ -0,0 +1,128 @@
+
+// Task #1
+
+// const createNewTeam = (nameOfTeam, nameOfDriver, dateBirthDriver, carBrand, teamSponsor, driversLicense) => {
+//     return {
+//         team: nameOfTeam,
+//         name: nameOfDriver,
+//         date: dateBirthDriver,
+//         car:  carBrand,
+//         sponsor: teamSponsor,
+//         license: driversLicense,
+//         isCreated: true,
+//     }
+// }
+
+// const newTeam = createNewTeam("Hillel", "Dmytro", 1992, "BMW", "Fil Dmytro", "confirm");
+
+// Task #2
+
+// let userInfo = {
+//     login: "",
+//     password: "",
+//     city: "",
+//     country: "",
+//     gender: "",
+//     age: "",

```

Comment:
Better to have them as `null` since it is the correct representation for empty value

```suggestion
//     login: null,
//     password: null,
//     city: null,
//     country: null,
//     gender: null,
//     age: null,

```